glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/minishift/minishift
 import:
 - package: github.com/docker/machine
-  version: 0.9.0
+  version: 0.11.0
```

Comment:
Wau, I am also surprised. I seem to recall having several issues when trying this some time back.